manifest.json

```diff
@@ -82,6 +112,19 @@
       "required" : true
     },
     {
+      "type": "string",
+      "format": "title",
+      "name": "db_info_staticip",
+      "title": "Static IP Addresses"
+    },
+    {
+      "name": "db_staticip_hero",
+      "type": "string",
+      "headerTemplate": "If the firewall of your database server requires to whitelist IP addresses, you can install the enterprise version of this connector. Here is how to: Go to the tab 'Advanced' and change the Ship URL to `https://hull-sql-enterprise.connectors.hullapp.net/` and save your changes. That's it!<br><br>The enterprise connector has the following static IP addresses:<br>- 34.228.67.206<br>- 34.195.105.157<br>- 52.1.26.74<br>- 34.230.113.192",
```

Comment:
Let's not communicate about those static IPs yet as they do not belong to us and will probably change at the end of the month if we move this off of the heroku private space.